cpp/cmake_modules/DefineOptions.cmake

```diff
@@ -405,6 +404,14 @@ takes precedence over ccache if a storage backend is configured" ON)
                 ARROW_IPC
                 ARROW_PARQUET)
 
+  define_option(ARROW_ACERO
+                "Build the Arrow Acero Engine Module"
+                OFF
+                DEPENDS
+                ARROW_COMPUTE
+                ARROW_IPC
+                ARROW_PARQUET)
```

Comment:
Why do we depend on IPC and parquet here?  Is it just for testing I wonder?  I think it's probably ok but I didn't have that dependency in my mind.